ACE/ace/SSL/SSL_Context.cpp

```diff
@@ -274,39 +254,8 @@ ACE_SSL_Context::set_mode (int mode)
     case ACE_SSL_Context::SSLv23:
       method = ::SSLv23_method ();
       break;
-    case ACE_SSL_Context::TLSv1_client:
```

Comment:
According to the OpenSSL/master man page for [SSL_CTX_new](https://www.openssl.org/docs/manmaster/ssl/SSL_CTX_new.html) all of the `TLSv1*` functions removed in this hunk exist in OpenSSL and `SSLv23_method()` added below is deprecated in favour of the `TLSv1*` ones.